app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreen.kt

```diff
@@ -0,0 +1,267 @@
+                            .fillMaxWidth()
+                            .padding(all = dimensions().spacing16x)
+                    ) {
+                        AnimatedContent(connectionStatus) {
+                            if (connectionStatus.isConnectionRequestPending) {
+                                WireSecondaryButton(
+                                    text = stringResource(R.string.label_cancel_request),
+                                    onClick = onCancelConnectionRequest
+                                )
+                            } else {
+                                WirePrimaryButton(
+                                    text = stringResource(R.string.label_connect),
+                                    onClick = onSendConnectionRequest,
+                                    leadingIcon = {
+                                        Icon(
+                                            painter = painterResource(id = R.drawable.ic_add_contact),
+                                            contentDescription = stringResource(R.string.content_description_right_arrow),
+                                            modifier = Modifier.padding(dimensions().spacing8x)
+                                        )
+                                    }
+                                )
+                            }
+                        }
+                    }
+                } else {
+                    Divider()
+                    Row(
+                        horizontalArrangement = Arrangement.Center,
+                        verticalAlignment = Alignment.CenterVertically,
+                        modifier = Modifier
+                            .height(dimensions().groupButtonHeight)
+                            .fillMaxWidth()
+                            .padding(all = dimensions().spacing16x)
+                    ) {
+                        WirePrimaryButton(
+                            text = stringResource(R.string.label_open_conversation),
+                            onClick = onOpenConversation,
+                        )
+                    }
+                }
+            }
+        }
+    }
+}
+
+@Composable
+private fun UserDetailInformation(
+    title: String,
+    value: String,
+    onCopy: (String) -> Unit
+) {
+    RowItemTemplate(
+        title = {
+            Text(
+                style = MaterialTheme.wireTypography.subline01,
+                color = MaterialTheme.wireColorScheme.labelText,
+                text = title.uppercase()
+            )
+        },
+        subtitle = {
+            Text(
+                style = MaterialTheme.wireTypography.body01,
+                color = MaterialTheme.wireColorScheme.onBackground,
+                text = value
+            )
+        },
+        actions = { CopyButton(onCopyClicked = { onCopy("$value copied") }) },
+        onRowItemClicked = { },
+        onRowItemLongClicked = { }
```

Comment:
Does this mean that these clicks will be executed, so the ripple animation will be visible, but no action will occur after the click?